objects/basic/set.php

```diff
@@ -76,6 +76,21 @@ function iconText() {
 		else
 			return l_t('No orders submitted!');
 	}
+	/*
+		Leaving all possible phases in place for clarity on the possible options to make future changes easier. 

```

Comment:
In general, I'm not a fan of extra code because we think we'll need it later - I think it's better to keep the code cleaner - and that short, clean code is easier to change.

As an example, spliting them out because of future possible changes introduces several places to change if we want to change the lock icon. I think it's better to only split when we need to.

Reply:
Okay, so you mean removing the case and just always having this function returning the anon information?